ui/src/localStorage.js

```diff
@@ -12,8 +14,22 @@ export const loadLocalStorage = errorsQueue => {
       console.log(errorText) // eslint-disable-line no-console
       errorsQueue.push(errorText)
 
-      window.localStorage.removeItem('state')
-      return {}
+      if (!state.dashTimeV1) {
```

Comment:
Can state be undefined?

Reply:
The existence of the state key can should be guaranteed by the if block just about it